tools/binary-compatibility/reference-public-api/reaktive.txt

```diff
@@ -207,6 +207,33 @@ public final class com/badoo/reaktive/disposable/VariousJvm {
 	public static final fun Disposable (Lkotlin/jvm/functions/Function0;)Lcom/badoo/reaktive/disposable/Disposable;
 }
 
+public abstract interface class com/badoo/reaktive/disposable/scope/DisposableScope : com/badoo/reaktive/disposable/Disposable {
```

Comment:
Maybe extract to separate module? It is not a core functionality.

Reply:
Perhaps, but I find it as a replacement for most of the `CompositeDisposable` use cases. So you will need it quite frequently. We could move `DisposableWrapper`, `CompositeDisposable` and `DisposableScope` later to a separate module. What do you think?

Reply:
If you want to use it inside operators, then it is fine to have it in main module.

Reply:
I think it's fine in the main module.